src/components/description_list/description_list_description.styles.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { css } from '@emotion/react';
+import {
+  euiFontSize,
+  euiBreakpoint,
+  logicalTextAlignCSS,
+  logicalCSS,
+} from '../../global_styling';
+import { UseEuiTheme } from '../../services';
+import { euiTitle } from '../title/title.styles';
+
+export const euiDescriptionListDescriptionStyles = (
+  euiThemeContext: UseEuiTheme
+) => {
+  const { euiTheme } = euiThemeContext;
+
+  const columnDisplay = `
+    ${logicalCSS('width', '50%')} // Flex-basis doesn't work in IE with padding
+    ${logicalCSS('padding-left', euiTheme.size.s)}
+    &:not(:first-of-type) {
+      ${logicalCSS('margin-top', euiTheme.size.base)}
+    }
+  `;
+
+  return {
+    euiDescriptionList__description: css``,
+
+    // Types
+    row: css``,
+    column: css`
+      ${columnDisplay}
+    `,
+    responsiveColumn: css`
+      ${euiBreakpoint(euiThemeContext, ['xs', 's'])} {
+        ${logicalCSS('width', '100%')}
+        padding: 0;
+      }
+      ${euiBreakpoint(euiThemeContext, ['xl'])} {
+        ${columnDisplay}
+      }
```

Comment:
I'm confused by this breakpoint selector - shouldn't this be m to xl?
```suggestion
      ${euiBreakpoint(euiThemeContext, ['m', 'xl'])} {
        ${columnDisplay}
      }
```

Reply:
Thank you for the catch!